unlock-app/src/components/interface/locks/CheckoutUrl/description.ts

```diff
@@ -0,0 +1,35 @@
+import { PaywallConfig, PaywallConfigLock } from '~/unlockTypes'
+
+type Description = Partial<
+  Record<keyof PaywallConfig | keyof PaywallConfigLock, string>
+>
+
+export const PaywallDescriptions: Description = {
```

Comment:
Is there a way to include these in the zod schema?

Reply:
yes! ill move it now into zod!

Reply:
https://github.com/unlock-protocol/unlock/pull/9993